__tests__/app.test.js

```diff
@@ -163,3 +163,37 @@ describe("GET: /api/users", () => {
     });
   });
 });
+
+describe("GET: /api/reviews", () => {
+  test("200:  returns an array of review objects, each of which should have the properties of owner, title review_id, category, review_img_url, created_at, votes, review_body, designer, comment_count", () => {
+    return request(app)
```

Comment:
Your assertion doesn't really match what you are describing here. You specify that each object has the right keys but explicitly check the first one. Why should the first one be this review in particular?